hasura/metadata/actions.graphql

```diff
@@ -222,7 +226,6 @@ input UpdateCircleInput {
   update_webhook: Boolean
   fixed_payment_token_type: String
   fixed_payment_vault_id: Int
-  show_pending_gives: Boolean
```

Comment:
why get rid of this in this PR?

Reply:
These were generated by the generate script as discussed, fixed manually for now with https://github.com/coordinape/coordinape/pull/1687/commits/9231d3344b1ebd0eda28693a8f9ab06b9673b416 and https://github.com/coordinape/coordinape/pull/1687/commits/290da405d9db9eec684d05f7c59634c51d5de769